src/TConsole.h

```diff
@@ -241,8 +241,6 @@ class TConsole : public QWidget
     int mDisplayFontSize = 14;
     QFont mDisplayFont = QFont(mDisplayFontName, mDisplayFontSize, QFont::Normal);
     int mEngineCursor = -1;
-    TChar mFormatBasic;
-    TChar mFormatSystemMessage;
```

Comment:
Whilst checking for `TChar` instances I found that this pair are never used in the rest of the code-base - so can be whipped out...